apps/aeprimop/src/aeprimop.erl

```diff
@@ -507,6 +509,12 @@ inc_account_nonce({Pubkey, Nonce, Force}, #state{} = S) ->
                 _ ->
                     Account1 = aec_accounts:set_nonce(Account, Nonce),
                     put_account(Account1, S1)
+            end;
+        false when DryRun andalso AccType == generalized ->
```

Comment:
What if we are in Dryrun and AccType == basic?

Reply:
Then we do the same checks as if we are not dry-running.